apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts

```diff
@@ -339,7 +339,7 @@ for (const { swap, xrayTicket } of tooLowAmountForQuoteSwaps) {
           await app.swap.verifySwapAmountErrorMessageIsDisplayed(
             electronApp,
             swap.accountToDebit,
-            "No quotes available.",
+            "Not enough balance, including network fee.",
```

Comment:
if previous message isn't relevant anymore, I guess you can remove the last check all together (supposing our partner return quotes in **any** case)

Reply:
No, partner will return no quotes in this case.

Reply:
then my comment is relevant, you can delete next set of instructions all together

Reply:
resolving, error priority updated on PTX level (LGTM to have this one now)